find-updates.py

```diff
@@ -296,7 +296,12 @@ def deploy_bluesky(client, entry, keys, hashtag):
 
     # Add the text to the textbuilder
     tb.text(message)
-    tb.link(entry["title"], entry["url"])
```

Comment:
You might want to do something like:

```python
title = (entry.get('title') or entry.get('name') or entry.get('url') or {}).get("url")
```
And choose a reasonable default if there is no url.